packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -274,6 +284,126 @@ LiveLabel.defaultProps = {
   dir: 'ltr',
 };
 
+/*
+ *  Index Alsos
+ */
+const StyledIndexAlsos = styled.div`
```

Comment:
Can we please split this file up into different files now, its got wayyyy to big 🙈

Reply:
Done